src/fonduer/candidates/mentions.py

```diff
@@ -535,22 +526,22 @@ def __init__(self, mention_classes, mention_spaces, matchers, **kwargs):
 
         super(MentionExtractorUDF, self).__init__(**kwargs)
 
-    def apply(self, context, clear, **kwargs):
+    def apply(self, doc, clear, **kwargs):
         """Extract mentions from the given Context.
 
         :param context: A document to process.
```

Comment:
Can you make sure these changed parameter names are also updated in the docstrings?